docs/tutorials/intro/executing_tasks.ipynb

```diff
@@ -0,0 +1,381 @@
+      "                                 \u001b[0m'mt-0/0/0'\u001b[0m,\n",
+      "                                 \u001b[0m'sp-0/0/0'\u001b[0m,\n",
+      "                                 \u001b[0m'ge-0/0/1'\u001b[0m,\n",
+      "                                 \u001b[0m'ge-0/0/2'\u001b[0m,\n",
+      "                                 \u001b[0m'.local.'\u001b[0m,\n",
+      "                                 \u001b[0m'dsc'\u001b[0m,\n",
+      "                                 \u001b[0m'gre'\u001b[0m,\n",
+      "                                 \u001b[0m'ipip'\u001b[0m,\n",
+      "                                 \u001b[0m'irb'\u001b[0m,\n",
+      "                                 \u001b[0m'lo0'\u001b[0m,\n",
+      "                                 \u001b[0m'lsi'\u001b[0m,\n",
+      "                                 \u001b[0m'mtun'\u001b[0m,\n",
+      "                                 \u001b[0m'pimd'\u001b[0m,\n",
+      "                                 \u001b[0m'pime'\u001b[0m,\n",
+      "                                 \u001b[0m'pp0'\u001b[0m,\n",
+      "                                 \u001b[0m'ppd0'\u001b[0m,\n",
+      "                                 \u001b[0m'ppe0'\u001b[0m,\n",
+      "                                 \u001b[0m'st0'\u001b[0m,\n",
+      "                                 \u001b[0m'tap'\u001b[0m,\n",
+      "                                 \u001b[0m'vlan'\u001b[0m]\u001b[0m,\n",
+      "             \u001b[0m'model'\u001b[0m: \u001b[0m'FIREFLY-PERIMETER'\u001b[0m,\n",
+      "             \u001b[0m'os_version'\u001b[0m: \u001b[0m'12.1X47-D20.7'\u001b[0m,\n",
+      "             \u001b[0m'serial_number'\u001b[0m: \u001b[0m'bcc9ff721778'\u001b[0m,\n",
+      "             \u001b[0m'uptime'\u001b[0m: \u001b[0m10477\u001b[0m,\n",
+      "             \u001b[0m'vendor'\u001b[0m: \u001b[0m'Juniper'\u001b[0m}\u001b[0m}\u001b[0m\n",
+      "\u001b[0m\n",
+      "\u001b[0m"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "AggregatedResult: print_result"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from brigade.plugins.tasks import networking\n",
+    "\n",
+    "cmh_spines = brg.filter(site=\"bma\", role=\"spine\")\n",
+    "result = cmh_spines.run(networking.napalm_get,\n",
+    "                        getters=[\"facts\"])                \n",
+    "cmh_spines.run(text.print_result,\n",
+    "              num_workers=1,\n",
+    "              data=result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pretty much the same pattern, just different task on different devices.\n",
+    "\n",
+    "## What is a task\n",
+    "\n",
+    "Let's take a look at what a task is. In it's simplest form a task is a function that takes at least a [Task](../../ref/api/task.html#brigade.core.task.Task) object as argument. For instance:\n",
```

Comment:
Should work (for the broken test) if changed to "../../ref/api/task.rst#brigade.core.task.Task"